web/modules/custom/custom_weather/custom_weather.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Custom Weather
```

Comment:
You already specified package: custom, why do you need that in the name as well?
Also, name could be a little bit more descriptive.